docs/Doxyfile

```diff
@@ -0,0 +1,21 @@
+DOXYFILE_ENCODING      = UTF-8
+
+PROJECT_NAME           = "OpenTelemetry C++ API/SDK"
+
+FILE_PATTERNS          = *.h *.cc
+
+GENERATE_LATEX         = NO
+
+GRAPHICAL_HIERARCHY    = NO
+
+INPUT                  = ../README.md ../CONTRIBUTING.md ../api/include ../sdk/include ../sdk/src
```

Comment:
Is there a reason for `../sdk/src` to be in that list? I think it shouldn't be there.

Reply:
No, actually I am not quite sure what should be included in the documentation. I will confirm this and update later. Thanks for pointing this out!